grails-app/views/order/_orderItemForm.gsp

```diff
@@ -0,0 +1,46 @@
+<tr>
+    <td class="middle">
+    </td>
+    <td>
+        <g:autoSuggest id="product" name="product" jsonUrl="${request.contextPath }/json/findProductByName?skipQuantity=true&supplierId=${order?.originParty?.id}"
+                       styleClass="text large"/>
+    </td>
+    <td class="middle center">
+        <select id="productSupplier" name="productSupplier.id"></select>
+    </td>
+    <td class="middle center">
+        <div id="supplierCode"></div>
+    </td>
+    <td class="middle center">
+        <div id="manufacturer"></div>
+    </td>
+    <td class="middle center">
+        <div id="manufacturerCode"></div>
+    </td>
+    <td class="middle center">
+        <input type="number" id="quantity" required="true" name='quantity' value="" size="10" class="text" />
```

Comment:
Let's be consistent with single and double quotes and lets treat name equally, i think this property want to have double quote too.